src/server/accessors/Notifier.ts

```diff
@@ -30,6 +35,26 @@ export class Notifier implements INotifier {
         await this.msgBridge.doNotifyRoom(room, message, this.appId);
     }
 
+    /**
+     * Sends a direct message to a user.
+     *
+     * @param {IUser} user - The user to send the message to.
+     * @param {IMessage} partialMsg - The partial message to send.
+     * @returns {Promise<void>} A Promise that resolves when the message has been sent.
+     */
+    public async notifyDM(user: IUser, partialMsg: Omit<IMessage, 'room'>): Promise<void> {
+        const sender = partialMsg.sender || (await this.userBridge.doGetAppUser(this.appId));
+        const dmRoom = (await this.roomBridge.doGetDirectByUsernames([user.username, sender.username], this.appId)) || (await this.createDMRoom(user, sender));
+
+        const message: IMessage = {
+            ...partialMsg,
+            room: dmRoom,
+            sender,
+        };
+
+        await this.msgBridge.doCreate(message, this.appId);
```

Comment:
also, should this not call doNotifyRoom ? Notifier is only for ephemeral messages and doCreate would create a normal message ..

Reply:
Thank you for the review, Deb! Actually, no, this is a method to be used by the Apps to notify users via a DM.

Reply:
okay but the thing is all notifier methods are ephemeral messages and that's what I, at least associate the accessor to function as.

At least if this has to be under notifier accessor, could the method name not use the `notify` keyword as to distinguish between ephemeral and persistent messages?

I'm saying this as a potential consumer of this api, that it can be confusing.

Reply:
Ah, interesting, but won't the `notifyDirectMessage` already explain that it will be a DM? I mean, if we remove the `notify`, it might create further confusion because it was designed to notify the user via a DM, or else there would be a thought: _Does it perhaps a new helper to let me easily create DMs_. And this might lead to missing the actual use-case. 

Do we need to update perhaps the `jsDoc`? I tried to keep it as informative as possible, but please do let me know if it is not able to deliver the correct message. I would appreciate it.

Thank you!